### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by ankitjain28may**

### CODE REVIEW

**Comment by ankitjain28may** [comment]
Can you please add the #orbtoberfest label?

**Comment by markstonehouse** [comment]
This is still a grammatical error. It should read "Notification from CircleCI. A critical test has failed." @ankitjain28may

File: src/@orb.yml [hunk 1]
```diff
@@ -61,7 +61,7 @@ commands:
       body:
         description: The Body parameter includes the full text of the message you want to send, limited to 1600 characters.
         type: string
-        default: "Notification from CircleCI. A critical test as failed."
+        default: "Notification from CircleCI. A critical test is failed."
```
Comment (gep13) [inline src/@orb.yml]:
```suggestion
        default: "Notification from CircleCI. A critical test has failed."
```

**Review by gep13 — CHANGES_REQUESTED** [review]
As suggested by @markstonehouse

**Comment by ankitjain28may** [comment]
Sorry, my bad, Thanks updated :)

**Comment by ankitjain28may** [comment]
@gep13 Any updates?

**Review by gep13 — APPROVED** [review]
LGTM!

**Comment by gep13** [comment]
@ankitjain28may I am not in a position to merge this PR, I only reviewed it.

**Review by lokst — APPROVED** [review]
LGTM

**Review by KyleTryon — APPROVED** [review]

**Comment by KyleTryon** [comment]
Thank you everyone. Merging. This will appear in the next major update.

**Pull request merged**